.github/workflows/windows.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           node-version: 18
 
       - name: Download cuda
-        run: azcopy.exe cp --recursive "https://lotusscus.blob.core.windows.net/models/cuda_sdk/v12.2" cuda_sdk
+        run: azcopy.exe cp --recursive "https://lotusscus.blob.core.windows.net/models/cuda_sdk/v12.3" cuda_sdk
```

Comment:
Did you copy cudnn files and zlib.dll to this folder?

Reply:
I copied zlibwapi.dll from v12.2/bin to v12.3/bin. Cudnn files are in both v12.2 and v12.3's `bin`, `lib` and `include` folders

Reply:
Good. Thank you!